recipes/arrays.md

```diff
@@ -64,7 +64,7 @@ items.unshift(42);
 ```dart
 final length = items.length;
 
-final newItems = items..addAll(otherItems);
+final newItems = items.addAll(otherItems);
```

Comment:
this is not a typo but rather [cascade notation](https://www.dartlang.org/guides/language/language-tour#cascade-notation-)